packages/block-editor/src/components/inserter/menu.native.js

```diff
@@ -115,6 +116,37 @@ export class InserterMenu extends Component {
 		this.setState( { numberOfColumns, itemWidth, maxWidth } );
 	}
 
+	getItems() {
```

Comment:
Could we add a comment here explaining why we need to pre-process the items list? (to add the paste option)

Reply:
Of course, I added a comment 👍